Classes/ViewHelpers/Pipe/EmailViewHelper.php

```diff
@@ -59,11 +52,15 @@ protected static function preparePipeInstance(
             $body = $renderChildrenClosure();
         }
         /** @var EmailPipe $pipe */
-        $pipe = GeneralUtility::makeInstance(ObjectManager::class)->get(EmailPipe::class);
+        $pipe = GeneralUtility::makeInstance('TYPO3\\CMS\\Extbase\\Object\\ObjectManager')
```

Comment:
This changed line is not necessary - class names should be imported and `::class` used.